sections/about/BoardSection.jsx

```diff
@@ -13,31 +15,72 @@ export const BoardSection = ({
       <div>
         <Title>{board}</Title>
         <p>{boardDescription}</p>
-        {boardMembers.map((member) => (
-          <Card
-            key={`About__Page__Card__${member.name}`}
-            thumbnail={member.avatar}
-            title={member.name}
-            imageWidth={300}
-            cardHeight={300}
-            component={
-              <div>
-                <Text>{member.name}</Text>
-              </div>
-            }
-          />
-        ))}
+        <Board>
+          {boardMembers.map((member) => (
+            <SCard
+              key={`About__Page__Card__${member.name}`}
+              thumbnail={member.avatar}
+              title={member.name}
+              imageWidth={member.imageWidth}
+              imageHeight={member.imageHeight}
+              cardHeight={member.cardHeight}
+              contentPadding={"5px"}
+              component={
+                <PositionWrapper>
+                  <Name fontcolor={member.fontColor}>{member.name}</Name>
+                  <Position fontcolor={member.fontColor}>{member.position}</Position>
+                </PositionWrapper>
+              }
+            />
+          ))}
+        </Board>
       </div>
     </Wrapper>
   );
 };
 
 const Wrapper = styled.div`
-  padding: 0 5%;
+  padding: 0 7%;
+  margin-bottom: 5vh;
 `;
 const Title = styled.h2`
   ${({ theme }) => `
         font-family: ${theme.font.kumbh};
         font-weight: 700;
     `};
 `;
+const Name = styled.h3`
+  ${({ theme, fontcolor }) => `
+        font-family: ${theme.font.kumbh};
+        font-weight: 700;
+        font-size: 1.4rem;
+        color: ${fontcolor ? fontcolor : baseTheme.colors.navy};
+    `};
+`;
+const Position = styled.p`
+  ${({ theme, fontcolor }) => `
+        font-family: ${theme.font.kumbh};
+        color: ${fontcolor ? fontcolor : baseTheme.colors.navy};
+        font-size: 1.1rem!important;
+        margin-top: -15px;
+    `};
+`;
+const PositionWrapper = styled.div`
+  text-align: center;
+`;
+const Board = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 5vh;
+  ${media(
+    "tablet",
+    `
+            flex-direction: column;
+        `
```

Comment:
I would make this breakpoint at mobile instead of tablet. The screen at tablet is too wide for singe column cards imo. 
- if breakpoint = "mobile" (current), on iPad (768px width) 
![screenshot_20210803_020358](https://user-images.githubusercontent.com/65516245/127965990-b113b645-d7cf-4e61-bd68-0e61b65a5771.png)
- if breakpoint = "mobile", on iPad (768px width) 
![screenshot_20210803_020341](https://user-images.githubusercontent.com/65516245/127965992-b8488de0-b960-4e95-97a2-8a73a3ce5ff8.png)